libraries/default.rb

```diff
@@ -39,6 +39,15 @@ def get_seed_file
       node.fetch(:cluster_seed_file, 'cluster_seed.tgz')
     end
 
+    def on_aws?
+      if node['vagrant_environment'] == true
+        Chef::Log.info "deploying to a vagrant cluster"
```

Comment:
oops, I issued this against the wrong branch. I've opened #71 against the correct branch.